tpp-app/src/settings/Settings.js

```diff
@@ -1,18 +1,160 @@
+        <View>
+            <Text style={styles.heading}>Notifications</Text>
+            <Card wrapperStyle={styles.rowContainer} containerStyle={[styles.dropShadow, styles.card]}>
+                    <Text>Remind me to log period</Text>
+                    <Switch
+                        onValueChange={props.togglePeriodSwitch}
+                        value={props.remindPeriodEnabled}
+                    />
+            </Card>
+            <Card wrapperStyle={styles.rowContainer} containerStyle={[styles.dropShadow, styles.card]}>
+
+                <Text> Remind me to log symptoms</Text>
                 <Switch
-                    onValueChange={toggleSwitch}
-                    value={isEnabled}
+                    onValueChange={props.toggleSymptomsSwitch}
+                    value={props.remindSymptomsEnabled}
                 />
             </Card>
         </View>
     )
 }
+
+const PreferenceButton = (props) => {
+    return (
+    <View style = {styles.horizontalCenteredColumn}>
+        <TouchableOpacity style={[styles.dropShadow, styles.preferenceButton]}>
+                <Image
+                    source={props.source}
+                />
+        </TouchableOpacity>
+        <Text>{props.cardName}</Text>
+
+    </View>
+    );
+}
+
+const Preferences = (props) => {
+    return (
+        <View>
+            <Text style={styles.heading}>Tracking Preferences </Text>
+            <View style={styles.preferences}>
+                <PreferenceButton source={FlowIcon} cardName="Flow"/>
+                <PreferenceButton source={MoodIcon} cardName="Mood"/>
+                <PreferenceButton source={SleepIcon} cardName="Sleep"/>
+                <PreferenceButton source={CrampsIcon} cardName="Cramps"/>
+                <PreferenceButton source={ExerciseIcon} cardName="Exercise"/>
+            </View>
+
+        </View>
+
+    );
+
+}
+
+
+export default function Settings () {
+    const [cycleLength, setCycleLength] = useState(35);
+    const [periodLength, setPeriodLength] = useState(5);
+    const [remindPeriodEnabled, setRemindPeriodEnabled] = useState(true);
+    const togglePeriodSwitch = () => setRemindPeriodEnabled(!remindPeriodEnabled);
+    const [remindSymptomsEnabled, setRemindSymptomsEnabled] = useState(true);
+    const toggleSymptomsSwitch = () => setRemindSymptomsEnabled(!remindSymptomsEnabled);
+
+    return (
+        <View style={styles.container}>
+            <Stats cycleLength={cycleLength} periodLength={periodLength}></Stats>
+            <Preferences/>
+            <Notifications
+                remindPeriodEnabled={remindPeriodEnabled}
+                remindSymptomsEnabled={remindSymptomsEnabled}
+                togglePeriodSwitch={togglePeriodSwitch}
+                toggleSymptomsSwitch={toggleSymptomsSwitch}
+            />
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    rowContainer: {
+        flexDirection: 'row',
+        justifyContent: 'space-between'
```

Comment:
Try adding an attribute `alignItems` to vertically center the text and switch with each other. ([attribute description here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#align-items))